src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1067,10 +1069,33 @@ private List<PackageItemRootViewModel> BindParentToChild(Dictionary<string, Pack
             }
 
             // Only add the folder items, they contain the files
-            updatedItems = items.Values.Where(x => !x.isChild).ToList();
+            var updatedItems = GetRootItems(items);
             return updatedItems;
         }
 
+        private List<PackageItemRootViewModel> GetRootItems(Dictionary<string, PackageItemRootViewModel> items)
+        {
+            var rootItems = items.Values.Where(x => !x.isChild).ToList();
+            if (!rootItems.Any()) return rootItems;
+
+            var root = new PackageItemRootViewModel(CurrentPackageDirectory);
+            var updatedItems = new List<PackageItemRootViewModel>();
+            //check each root item and create any missing connections
+            foreach (var item in rootItems)
+            {
+                var itemDir = new DirectoryInfo(item.DirectoryName);
+                if (!itemDir.Parent.FullName.Equals(CurrentPackageDirectory))
+                {
+                    root.AddChild(item);
+                }
+                else
+                {
+                    root.ChildItems.Add(item);
+                }
+            }
+            return root.ChildItems.ToList();
+        }   
+
         /// <summary>
         /// Test if path2 is subpath of path1
         /// If it is, make sure all the intermediate file paths are created as separte PackageItemRootViewModel
```

Comment:
I know this is existing, but spelling error and I don't think this second line is implemented in this function.